internal/serve/graphql/schema/operation.graphqls

```diff
@@ -0,0 +1,19 @@
+# GraphQL Operation type - represents a blockchain operation
+# Operations are the individual actions within a transaction
+type Operation{
+  id:              Int64!
+  operationType:   OperationType!
+  operationXdr:    String!
+  ledgerCreatedAt: Time!
+  ingestedAt:      Time!
+  
+  # GraphQL Relationships - these fields use resolvers  
+  # Parent transaction - nullable because resolver might not find it
+  transaction: Transaction @goField(forceResolver: true)
```

Comment:
I'm not familiar enough with dataloaders and resolvers to understand, why wouldn't the resolver not find it?